hotfixes/4.3.15.cfg

```diff
@@ -0,0 +1,12 @@
+[buildout]
+
+hotfix-eggs =
+
```

Comment:
I've proposed constraint updates in https://github.com/jone/plone-hotfixes/pull/9